XCTestBootstrap/TestManager/FBTestManagerTestReporterTestCase.m

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBTestManagerTestReporterTestCase.h"
+
+@interface FBTestManagerTestReporterTestCase ()
+
+@property (nonatomic, copy) NSString *method;
+@property (nonatomic, copy) NSString *testClass;
+@property (nonatomic, readonly) NSString *statusDescription;
+@property (nonatomic) NSMutableArray<FBTestManagerTestReporterTestCaseFailure *> *mutableFailures;
+
+@end
+
+@implementation FBTestManagerTestReporterTestCase
+
++ (instancetype)withTestClass:(NSString *)testClass method:(NSString *)method
+{
+  return [[self alloc] initWithTestClass:testClass method:method];
+}
+
+- (instancetype)initWithTestClass:(NSString *)testClass method:(NSString *)method
+{
+  self = [super init];
+  if (!self) {
+    return nil;
+  }
+
+  _testClass = [testClass copy];
+  _method = [method copy];
+  _mutableFailures = [NSMutableArray array];
+
+  return self;
+}
+
+- (NSArray<FBTestManagerTestReporterTestCaseFailure *> *)failures
+{
+  return [self.mutableFailures copy];
+}
+
+- (void)addFailure:(FBTestManagerTestReporterTestCaseFailure *)failure
+{
+  [self.mutableFailures addObject:failure];
+}
+
+#pragma mark -
+
+- (NSString *)description
+{
+  return [NSString stringWithFormat:@"TestCase %@ | Method %@ | Status %@ | Duration %f", self.testClass, self.method,
+                                    self.statusDescription, self.duration];
+}
+
+- (NSString *)statusDescription
```

Comment:
Add the inverse of the `+[FBTestManagerResultSummary statusForStatusString:]` method and call it from here.